packages/renderer/src/lib/image/PullImage.svelte

```diff
@@ -147,27 +205,36 @@ function requestFocus(element: HTMLInputElement) {
   </svelte:fragment>
 
   <div slot="content" class="space-y-6">
-    <div class="w-full">
+    <div class="w-full text-[var(--pd-content-card-text)]">
       <label for="imageName" class="block mb-2 font-bold text-[var(--pd-content-card-header-text)]"
         >Image to Pull</label>
-      <input
-        id="imageName"
-        class="w-full p-2 outline-none bg-[var(--pd-select-bg)] border-[1px] border-transparent border-b-[var(--pd-input-field-stroke)] rounded-sm text-[var(--pd-content-card-text)] placeholder:text-[color:var(--pd-input-field-placeholder-text)]"
-        type="text"
-        name="imageName"
-        disabled={pullFinished || pullInProgress}
-        on:input={event => validateImageName(event)}
-        on:keypress={event => {
-          if (event.key === 'Enter') {
-            pullImage();
-          }
-        }}
-        bind:value={imageToPull}
-        aria-invalid={imageNameInvalid !== ''}
-        placeholder="Image name"
-        aria-label="imageName"
-        required
-        use:requestFocus />
+      <div class="grid grid-cols-2 gap-4">
+        <Select
+          id="imageName"
+          name="imageName"
+          aria-label="imageName"
+          aria-invalid={imageNameInvalid !== ''}
+          disabled={pullFinished || pullInProgress}
+          placeholder="Registry name / Image name"
+          required
+          focused
+          debounceWait={debounceWaitSearch}
+          onChange={e => onChange(e.detail.value)}
+          onClear={() => onChange('')}
+          loadOptions={searchImages}>
+          <div slot="empty">
+            You can search an image in a specific registry<br />by typing <code>registry-name/query</code>
+          </div>
+        </Select>
+        <Select
+          class="grow"
+          items={imageTags}
+          disabled={!tagsEnabled || pullFinished || pullInProgress}
+          placeholder="Image tag"
```

Comment:
```suggestion
          placeholder="Image tag (optional)"
```

![image](https://github.com/user-attachments/assets/cccf68b2-ee61-4949-a161-321125db0bce)


what do you think?